src/mscorlib/shared/System/Runtime/InteropServices/MemoryMarshal.cs

```diff
@@ -23,5 +24,32 @@ public static class MemoryMarshal
         /// </remarks>
         public static Memory<T> AsMemory<T>(ReadOnlyMemory<T> readOnlyMemory) =>
             Unsafe.As<ReadOnlyMemory<T>, Memory<T>>(ref readOnlyMemory);
+        
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Why is the AggresiveInlining necessary?

I know that you got into habit of marking everything with AggresiveInlining - I do not think it is a good habit to follow.

Reply:
For this instance, I wanted to remain consistent with DangerousGetPinnableReference which already had it, presumable for good reason. I will try to run some experiments.

Reply:
ok

Reply:
Inlining in almost all cases made no difference.

The only case inlining helps is if the implementation is using `in` and the `Reference` property.